app/javascript/blacklight/index.js

```diff
@@ -2,14 +2,32 @@ import BookmarkToggle from './bookmark_toggle'
 import ButtonFocus from './button_focus'
 import FacetLoad from './facet_load'
 import Modal from './modal'
+import Bootstrap4Modal from './bootstrap4Modal'
 import SearchContext from './search_context'
 import Core from './core'
 
+// We keep configuration data for the modal in the Core.modal object.
+// Create lazily if someone else created first.
+if (Core.modal === undefined) {
+  Core.modal = {
+    modalSelector: '#blacklight-modal' // a Bootstrap modal div that should be already on the page hidden
+  }
+}
+
+Core.onLoad(function() {
+  let bootstrapModal;
+  if (typeof(jQuery) !== 'undefined' && jQuery.fn.tooltip.Constructor.VERSION) {
```

Comment:
```suggestion
  if (typeof(jQuery) !== 'undefined' && jQuery.fn.tooltip.Constructor.VERSION.match(/^4\./)) {
```

Reply:
Recognizing that apps might still pull in jquery (cf https://getbootstrap.com/docs/5.0/getting-started/javascript/#still-want-to-use-jquery-its-possible), maybe better to explicitly test the version number?

Reply:
Yes, I have just this situation:
<img width="256" alt="Screen Shot 2022-02-09 at 3 02 24 PM" src="https://user-images.githubusercontent.com/92044/153289512-089edd11-f988-4247-b0d8-7698967dba0a.png">